DocumentConverter.BusinessLogic/Classes/InternalFormat/InternalFormatRepository.cs

```diff
@@ -0,0 +1,27 @@
+﻿using DocumentConverter.Contracts.Interfaces.InternalFormat;
+using DocumentConverter.EF.Core.Models;
+using System.Linq;
+
+namespace DocumentConverter.BusinessLogic.Classes.InternalFormat
+{
+    public class InternalFormatRepository : IInternalFormatRepository
+    {
+        private readonly OrganizationsAndDocumentsContext _organizationsAndDocumentsContext;
+        public InternalFormatRepository(OrganizationsAndDocumentsContext organizationsAndDocumentsContext)
+        {
+            _organizationsAndDocumentsContext = organizationsAndDocumentsContext;
+        }
+        public bool FindOrganizationById(string id)
```

Comment:
Class name: **InternalFormat**Repository
Method name: Find**Organization**ById

This seems out of place